src/curvefit/solvers/solvers.py

```diff
@@ -32,8 +33,11 @@ def get_model_instance(self):
     def fit(self, data, x_init=None, options=None):
         raise NotImplementedError()
 
+    def clone(self):
```

Comment:
Maybe we should just subclass Solver from Prototype then?

Reply:
Yes, good catch... could have sworn I did that already. Going to push in next commit!

Reply:
@aksholokhov see commit 8b732ba